capstone-frontend/src/components/Home.jsx

```diff
@@ -0,0 +1,304 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom'
+import { CardDeck, Tabs, Tab, Card, Spinner, Button } from 'react-bootstrap';
+import ClubGridItem from './ClubGridItem.jsx'
+import { BookDescriptionOverlay } from "./BookDescriptionOverlay";
+import CreateList from './CreateList'
+import '../App.css';
+import '../styles/Home.css'
```

Comment:
nit: be more consistent here with semicolons

Reply:
Ok